samples/testing-frameworks/appium/client-side/python/base_test.py

```diff
@@ -40,14 +40,16 @@ def setUp(self):
         self.utils.log("Will save screenshots at: " + self.screenshot_dir)
 
         self.desired_capabilities_cloud = {}
-        self.desired_capabilities_cloud["bitbar_apiKey"] = bitbar_apiKey
-        self.desired_capabilities_cloud["bitbar_testrun"] = bitbar_testrun_name
-        self.desired_capabilities_cloud["bitbar_app"] = bitbar_app
+        self.desired_capabilities_cloud["appium_newCommandTimeout"] = new_command_timeout
         self.desired_capabilities_cloud["fullReset"] = False
         self.desired_capabilities_cloud["noReset"] = True
```

Comment:
Those are appium capabilities, so the should be prefixed with `appium:<caps name>` or wrapped in `appium:options`